.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,32 @@
+name: Build
+on:
+  push:
+    branches:
+      - main
```

Comment:
It's still `master` for this repo 😁

Reply:
Guess that is not a nitpick. I totally overlooked it.